src/senaite/core/listing/view.py

```diff
@@ -218,13 +218,15 @@ def before_render(self):
             subscriber.before_render()
 
     def get_listing_view_adapters(self):
-        """Returns subscriber adapters used to modify the listing behavior
+        """Returns subscriber adapters used to modify the listing behavior,
+        sorted from higher to lower priority
         """
         # Allows to override this listing by multiple subscribers without the
         # need of inheritance. We use subscriber adapters here because we need
         # different add-ons to be able to modify columns, etc. without
         # dependencies amongst them.
-        return subscribers((self, self.context), IListingViewAdapter)
+        adapters = subscribers((self, self.context), IListingViewAdapter)
+        return sorted(adapters, key=lambda ad: float(ad.get_priority_order()))
```

Comment:
However, maybe use better here `api.to_float` and make the priority order an attribute of the adapter?

Reply:
I thought about making the priority an attribute of the adapter, but at the same time I wanted this to be in the Interface signature. I finally decided for the latter, but a matter of taste.

Reply:
Attributes can also be part of the interface signature with `zope.interface.Attribute`, see https://zopeinterface.readthedocs.io/en/latest/README.html#interfaces

I think forcing this has the downside that everyone will use `0` as the priority, which makes this kind of useless.

Better make it optional and use a default of e.g. `1000`

Reply:
Good, did not know that!. Thanks, much better to use an attribute, indeed. Will open another PR for this.